internal/acceptance/grant_test.go

```diff
@@ -134,3 +137,33 @@ func TestUcAccGrantForIdChange(t *testing.T) {
 		ExpectError: regexp.MustCompile(`cannot create grant: Privilege abc is not applicable to this entity`),
 	})
 }
+
+func grantTemplateForUnderscoreChange(permission string) string {
+	return fmt.Sprintf(`
+	resource "databricks_storage_credential" "external" {
+		name = "cred-{var.STICKY_RANDOM}-hyphens"
+		aws_iam_role {
+			role_arn = "{env.TEST_METASTORE_DATA_ACCESS_ARN}"
+		}
+		comment = "Managed by TF"
+	}
+	
+	resource "databricks_grant" "cred" {
+		storage_credential = databricks_storage_credential.external.id
+		principal  = "{env.TEST_DATA_ENG_GROUP}"
+		privileges = ["%s"]
+	}
+	`, permission)
+}
+
+func TestUcAccGrantForUnderscoreChange(t *testing.T) {
+	unityWorkspaceLevel(t, step{
+		Template: grantTemplateForUnderscoreChange("ALL_PRIVILEGES"),
```

Comment:
it should suffice to just have a single step with `ALL PRIVILEGES`, the test would fail if there is still a diff shown in the plan

Reply:
Ok, updated